### ISSUE TITLE
archived field in project table does not have a default value

### ISSUE THREAD

**Issue by gauthier** [Block8/PHPCI#1237]
Which causes project addition to fail on MySQL servers with strict error reporting.

I had to alter the table this way to be able to create a project:

```sql
ALTER TABLE project CHANGE archived archived tinyint(1) NOT NULL DEFAULT 0;
```

**Comment by dancryer** [comment]
This was fixed Yesterday in v1.7.1. 👍 

https://github.com/Block8/PHPCI/releases/tag/1.7.1

**Comment by gauthier** [comment]
oh I'm sorry, it looks that I'm not fully up to date :)

thanks for having fixed it anyway!
